Jenkinsfile

```diff
@@ -137,6 +137,23 @@ pipeline {
         }
       }
     }
+    stage('Building packages') {
+      agent { label 'docker && linux && immutable' }
+      options { skipDefaultCheckout() }
+      environment {
+        HOME = "${env.WORKSPACE}"
+        PATH = "${env.PATH}:${env.WORKSPACE}/.local/bin"
+      }
+      steps {
+        deleteDir()
+        unstash 'source'
+        dir("${BASE_DIR}"){
+          sh script: 'pip3 install --user cibuildwheel', label: "Installing cibuildwheel"
+          sh script: 'mkdir wheelhouse', label: "creating wheelhouse"
+          sh script: 'cibuildwheel --platform linux --output-dir wheelhouse; ls -l wheelhouse'
+        }
```

Comment:
```suggestion
          stash allowEmpty: true, name: 'packages', includes: "path_to_packages/**", useDefaultExcludes: false
        }
```

Reply:
this store temporarily the packages in Jenkins, we can also archive them in Jenkins with the artifacts step

```
artifacts: "path_to_packages/**", onlyIfSuccessful: true)
```